packages/content-management/table_list/src/services.tsx

```diff
@@ -0,0 +1,191 @@
+ */
+export interface TableListViewKibanaDependencies {
+  /** CoreStart contract */
+  core: {
+    application: {
+      capabilities: {
+        advancedSettings?: {
+          save: boolean;
+        };
+      };
+      getUrlForApp: (app: string, options: { path: string }) => string;
+      currentAppId$: Observable<string | undefined>;
+      navigateToUrl: (url: string) => Promise<void> | void;
+    };
+    notifications: {
+      toasts: {
+        addDanger: (notifyArgs: { title: MountPoint; text?: string }) => void;
+      };
+    };
+  };
+  /**
+   * Handler from the '@kbn/kibana-react-plugin/public' Plugin
+   *
+   * ```
+   * import { toMountPoint } from '@kbn/kibana-react-plugin/public';
+   * ```
+   */
+  toMountPoint: (
+    node: React.ReactNode,
+    options?: { theme$: Observable<{ readonly darkMode: boolean }> }
+  ) => MountPoint;
+  /**
+   * The public API from the savedObjectsTaggingOss plugin.
+   * It is returned by calling `getTaggingApi()` from the SavedObjectTaggingOssPluginStart
+   *
+   * ```js
+   * const savedObjectsTagging = savedObjectsTaggingOss?.getTaggingApi()
+   * ```
+   */
+  savedObjectsTagging?: {
+    ui: {
+      getTableColumnDefinition: () => EuiTableFieldDataColumnType<UserContentCommonSchema>;
+      parseSearchQuery: (
+        query: string,
+        options?: {
+          useName?: boolean;
+          tagField?: string;
+        }
+      ) => {
+        searchTerm: string;
+        tagReferences: SavedObjectsFindOptionsReference[];
+        valid: boolean;
+      };
+      getSearchBarFilter: (options?: {
+        useName?: boolean;
+        tagField?: string;
+      }) => SearchFilterConfig;
+    };
+  };
+  /** The <FormattedRelative /> component from the @kbn/i18n-react package */
+  FormattedRelative: typeof FormattedRelative;
+}
+
+/**
+ * Kibana-specific Provider that maps to known dependency types.
+ */
+export const TableListViewKibanaProvider: FC<TableListViewKibanaDependencies> = ({
+  children,
+  ...services
+}) => {
+  const { core, toMountPoint, savedObjectsTagging, FormattedRelative } = services;
+
+  const getSearchBarFilters = useMemo(() => {
+    if (savedObjectsTagging) {
+      return () => [savedObjectsTagging.ui.getSearchBarFilter({ useName: true })];
+    }
+  }, [savedObjectsTagging]);
+
+  const searchQueryParser = useMemo(() => {
+    if (savedObjectsTagging) {
+      return (searchQuery: string) => {
+        const res = savedObjectsTagging.ui.parseSearchQuery(searchQuery, { useName: true });
+        return {
+          searchQuery: res.searchTerm,
+          references: res.tagReferences,
+        };
+      };
+    }
+  }, [savedObjectsTagging]);
+
+  return (
+    <RedirectAppLinksKibanaProvider coreStart={core}>
+      <TableListViewProvider
+        canEditAdvancedSettings={Boolean(core.application.capabilities.advancedSettings?.save)}
+        getListingLimitSettingsUrl={() =>
+          core.application.getUrlForApp('management', {
```

Comment:
Can we this extract this into a constant?

Reply:
This is extensively used accross Kibana. The day we rename the `management` app I expect a big search and replace 😊 

 
<img width="628" alt="Screenshot 2022-09-19 at 09 13 31" src="https://user-images.githubusercontent.com/2854616/190976012-62b827cf-47cb-43ae-b100-d97c24215f96.png">